src/steam-user/steam-user.controller.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Patch,
+  Param,
+  Delete,
+} from '@nestjs/common';
+import { SteamUserService } from './steam-user.service';
+import { CreateSteamUserDto } from './dto/create-steam-user.dto';
+import { UpdateSteamUserDto } from './dto/update-steam-user.dto';
+
+@Controller('steam')
+export class SteamUserController {
+  constructor(private readonly steamUserService: SteamUserService) {}
+
+  @Post()
+  create(@Body() createSteamUserDto: CreateSteamUserDto) {
+    return this.steamUserService.create(createSteamUserDto);
+  }
```

Comment:
This is a good endpoint as user can connect steam profile later in settings. That should always act like a connect, so you should use a current user id